extensions.json

```diff
@@ -515,13 +515,13 @@
   "johnpapa.winteriscoming": {
     "repository": "https://github.com/johnpapa/vscode-winteriscoming"
   },
-  "johnsoncodehk.volar": {
+  "Vue.volar": {
     "repository": "https://github.com/johnsoncodehk/volar",
     "location": "extensions/vscode-vue-language-features",
     "custom": [
       "npx pnpm install --no-frozen-lockfile --shamefully-hoist",
       "npx pnpm run build",
-      "cd ./extensions/vscode-vue-language-features && npx vsce package -o extension.vsix"
+      "cd ./extensions/vscode-vue-language-features && npx pnpm run prepack:browser && npx vsce package -o extension.vsix"
     ]
   },
   "johnsoncodehk.vscode-typescript-vue-plugin": {
```

Comment:
Extension id `johnsoncodehk.vscode-typescript-vue-plugin` is also changed to `Vue.vscode-typescript-vue-plugin`.